python/transcriber/vosk_transcriber.py

```diff
@@ -0,0 +1,76 @@
+import logging
```

Comment:
This script has to be installed into bin script with setup.py machinery. It is missing from the patch